### DISCUSSION TITLE
Statement of FSF board on election of Richard Stallman

### DISCUSSION THREAD

**Discussion by cavallium** [rms-support-letter/rms-support-letter.github.io#6685] [General]
Statement of FSF board on election of Richard Stallman:
https://www.fsf.org/news/statement-of-fsf-board-on-election-of-richard-stallman

RMS addresses the free software community:
https://www.fsf.org/news/rms-addresses-the-free-software-community

**Comment by franpoli** [comment]
This wise statement was not well received by the opposition, [which reiterates its call for ~~lynching~~ mob justice](https://github.com/rms-open-letter/rms-open-letter.github.io/commit/b5216c7ac7bb876186245833409627e575738ece).

Edit: I was indecently demonized by the [FSFE](https://mastodon.social/@kirschner/106097048958512607) for using the term "lynching", in order to avoid any confusion I replacing the term by "mob justice" that reflects better my thought.

**Reply by AnimusPEXUS** [reply]
on the other hand, now it is obvious, what lynching is the only competence they capable of: haters gonna hate.

**Reply by Aspie96** [reply]
Also without asking those who signed first.
At least one person dissented

**Reply by Aspie96** [reply]
https://github.com/rms-open-letter/rms-open-letter.github.io/pull/2556#issuecomment-818475723

**Comment by KOLANICH** [comment]
Oh shit.

A person who cannot even verbally defend himself obviously cannot defend FOSS. And IMHO it is very unwise to compromise with the kind of enemy that cannot be compromised with and which on each compromise will only increase pressure. I understand that Stallman was pushed by the rest of FSF, but if one cannot resist the pressure, he is a poor leader.

I guess a yet another "letter" may be needed, urging Stallman to do some self-training to restore his personal qualities.

![](https://lurkmore.so/images/1/17/Byd_myzhikom_clean.jpeg)

**Reply by AnimusPEXUS** [reply]
why do you think he compromised? he's basically said: "I admit it: I was rough, but I was right" in the form looking like compromise/excuse. :D

**Reply by KOLANICH** [reply]
He and FSF shouldn't have make these statements at all. No negotiations with blackmailers are acceptable. Once one has started negotiating with blackmailers, he is already almost defeated.

**Reply by AnimusPEXUS** [reply]
no, he done wise - he flipped the wave and made his accusers guilty in hatred and extremism spreading.

**Reply by 6r1d** [reply]
> no, he done wise - he flipped the wave and made his accusers guilty in hatred and extremism spreading.

If you apologize to a mob, they will only attack you more viciously from that point on.

Bryan Lunduke: [Stallman & The FSF Respond To The Mob!](https://open.lbry.com/@Lunduke:e/RMSFSFRespondToMob:2?r=GAaJ9s3EZNYqp61oaAXUcUzgEMPmhhf1)

**Reply by AnimusPEXUS** [reply]
>will only attack you more viciously from that point on

Stallman did right. He showed what he is human, and not like wildly-behaving masses. Also, I'll repeat, this apologize, like Stallman did, is not so simple. Stallman is wise man.

**Reply by Aspie96** [reply]
Did you read his apology?

He didn't apologize about anything he said nor did he admit guilt he does not have.

He just apologized about not being good at dealing with people which we all already know to be true

**Reply by 6r1d** [reply]
> He just apologized about not being good at dealing with people which we all already know to be true

Sure, and that makes sense. The only problem is that remains an "attack vector", but everything does.

**Comment by Aspie96** [comment]
Now 3 FSFs out of 4 support RMS (well FSFLA doesn't but its president does and FSFLA is not against him).

**Reply by 6r1d** [reply]
I'm happy to hear that.

**Reply by franpoli** [reply]
Maybe we could collect these statements, on the section of the wiki? I read the [FSF India Board Statement](https://fsf.org.in/news/board-statement-2021/).

**Reply by Aspie96** [reply]
Note: I was wrong. Oliva is not the president, just a board member.
